app/__init__.py

```diff
@@ -22,12 +19,10 @@ def create_app(config_name):
     application.wsgi_app = ProxyFix(application.wsgi_app)
     application.config.from_object(config[config_name])
 
-    for name in config_attrs(config[config_name]):
-        if name in os.environ:
-            application.config[name] = convert_to_boolean(os.environ[name])
```

Comment:
@minglis just copied this pattern you've deleted into the supplier app.  We should probably standardise how config is set up a bit across all the apps.